pip/flatpak-pip-generator

```diff
@@ -27,6 +27,8 @@ parser.add_argument('--cleanup', choices=['scripts', 'all'],
                     help='Select what to clean up after build')
 parser.add_argument('--requirements-file', '-r',
                     help='Specify requirements.txt file')
+parser.add_argument('--ignore-pkg', '-i', action='append',
+                    help='Ignore this package, e.g. if the package didn\'t upload source packages to PyPI')
```

Comment:
Also, we shouldn't allow for `--ignore-pkg` and `packages` at the same time since that makes no sense. Parameter help should be updated accordingly